importpub/gscholar.py

```diff
@@ -14,10 +21,23 @@ def get_publications(author_id):
     author_object = scholarly.fill(search_query, publication_limit=10)
     pub_list = []
 
+    processes = []
+    parent_connections = []
+
     for pub in author_object["publications"]:
-        current_pub = scholarly.fill(pub)
-        del current_pub["source"]
-        pub_list.append(current_pub)
+        parent_conn, child_conn = Pipe()
+        parent_connections.append(parent_conn)
+        process = Process(target=fill_pub, args=(pub,child_conn))
+        processes.append(process)
```

Comment:
nice job using IPC to get data from parent to child via pipes!
Another alternative would be using shared memory: https://docs.python.org/3/library/multiprocessing.shared_memory.html and locks, but this looks good.